### PR TITLE
Added Accepted and Opened Flaky Tests

### PR DESCRIPTION
**Pull request by priyanka-28**

### CODE REVIEW

File: pr-data.csv [hunk 1]
```diff
@@ -1814,8 +1814,8 @@ https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar
...
 https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-scripting-bundle/nifi-scripting-processors,org.apache.nifi.processors.script.TestScriptedPartitionRecord.testWhenMultiplePartitions,ID,,,
 https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-solr-bundle/nifi-solr-processors,org.apache.nifi.processors.solr.TestQuerySolr.testAttributes,ID,,,
-https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-splunk-bundle/nifi-splunk-processors,org.apache.nifi.processors.splunk.TestPutSplunkHTTP.testHappyPathWithCustomQueryParameters,ID,,,
-https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-splunk-bundle/nifi-splunk-processors,org.apache.nifi.processors.splunk.TestPutSplunkHTTP.testHappyPathWithCustomQueryParametersFromFlowFile,ID,,,
+https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-splunk-bundle/nifi-splunk-processors,org.apache.nifi.processors.splunk.TestPutSplunkHTTP.testHappyPathWithCustomQueryParameters,ID,Opened,https://github.com/apache/nifi/pull/6672/files,
```
Comment (darko-marinov) [inline pr-data.csv:1817]: Please remove `/files` from the PR Link.

File: pr-data.csv [hunk 2]
```diff
@@ -3646,8 +3646,8 @@ https://github.com/schemacrawler/SchemaCrawler,18d6c5fb79c8e27cdeafc317ee63b2783
 https://github.com/schemacrawler/SchemaCrawler,18d6c5fb79c8e27cdeafc317ee63b27837aa6fec,schemacrawler-utility,us.fatehi.utility.test.html.TableCellTest.td2,ID,InspiredAFix,https://github.com/schemacrawler/SchemaCrawler/pull/525,
 https://github.com/seata/seata,d334f85395887193f72fa57daea906a73a2da34e,common,io.seata.common.util.CollectionUtilsTest.encodeMap,ID,Accepted,https://github.com/seata/seata/pull/1802,
 https://github.com/seata/seata,d334f85395887193f72fa57daea906a73a2da34e,tm,io.seata.tm.api.transaction.TransactionInfoTest.testFieldGetSetFromJson,ID,Deleted,,https://github.com/seata/seata/commit/6c0c75939dedbf19c4a21a8ac7aa0fb98879a8fd
-https://github.com/sinch/robovm,ef091902377c00dc0fb2db87e8d79c8afb5e9010,compiler,org.robovm.compiler.config.ConfigTest.testWriteConsole,ID,,,
-https://github.com/sinch/robovm,ef091902377c00dc0fb2db87e8d79c8afb5e9010,compiler,org.robovm.compiler.config.ConfigTest.testWriteIOS,ID,,,
+https://github.com/sinch/robovm,ef091902377c00dc0fb2db87e8d79c8afb5e9010,compiler,org.robovm.compiler.config.ConfigTest.testWriteConsole,ID,Opened,https://github.com/robovm/robovm/pull/1272,
```
Comment (darko-marinov) [inline pr-data.csv:3649]: This PR link should be on `robovm/robovm` not `sinch/robovm`. In fact, these forked repos like `sinch/robovm` should not even be in `pr-data.csv`. Please remove these entries (as in delete them altogether not just add `Deleted`)?

**Review by darko-marinov — CHANGES_REQUESTED** [review]
Good changes but need to pass through the checker.

File: pr-data.csv (hunk 2 above)
Comment (priyanka-28) [reply pr-data.csv]: Thank you, updated!

**Pull request merged**